src/components/ClusterNodes/ClusterNodeContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { URLS } from '../../config'
+import { Redirect, Route, Switch } from 'react-router-dom'
+import ClusterList from './ClusterList'
+import NodeDetails from './NodeDetails'
+import NodeList from './NodeList'
+
+export default function ClusterNodeContainer() {
+    return (
+        <>
```

Comment:
remove empty/diamond tag as it's not required

Reply:
Done